scale/docs/rest/v6/scale_file.rst

```diff
@@ -0,0 +1,418 @@
+        "results": [ 
+            { 
+                "id": 465, 
+                "workspace": { 
+                    "id": 2, 
+                    "name": "Products" 
+                }, 
+                "file_name": "my_file.kml", 
+                "media_type": "application/vnd.google-earth.kml+xml", 
+                "file_size": 100, 
+                "data_type": [], 
+                "is_deleted": false, 
+                "url": "http://host.com/file/path/my_file.kml", 
+                "created": "1970-01-01T00:00:00Z", 
+                "deleted": null, 
+                "data_started": null, 
+                "data_ended": null, 
+                "geometry": null, 
+                "center_point": null, 
+                "countries": ["TCY", "TCT"], 
+                "last_modified": "1970-01-01T00:00:00Z", 
+                "file_path": "path/to/the/file.png",
+                "is_operational": true, 
+                "is_published": true, 
+                "has_been_published": true, 
+                "published": "1970-01-01T00:00:00Z", 
+                "unpublished": null, 
+                "source_started": "1970-01-01T00:00:00Z", 
+                "source_ended": "1970-01-02T00:00:00Z", 
+                "job": { 
+                    "id": 47 
+                }, 
+                "job_exe": { 
+                    "id": 49 
+                },
+                "job_output": "output_name_1",
+                "job_type": { 
+                    "id": 8, 
+                    "name": "kml-footprint", 
+                    "version": "1.0.0", 
+                    "title": "KML Footprint", 
+                    "description": "Creates a KML file.", 
+                    "category": "footprint", 
+                    "author_name": null, 
+                    "author_url": null, 
+                    "is_system": false, 
+                    "is_long_running": false, 
+                    "is_active": true, 
+                    "is_operational": true, 
+                    "is_paused": false, 
+                    "icon_code": "f0ac" 
+                }, 
+                "recipe": { 
+                    "id": 60 
+                }, 
+                "recipe_job": "kml-footprint",
+                "recipe_type": { 
+                    "id": 6, 
+                    "name": "my-recipe", 
+                    "version": "1.0.0", 
+                    "title": "My Recipe", 
+                    "description": "Processes some data", 
+                }, 
+                "batch": { 
+                    "id": 15, 
+                    "title": "My Batch", 
+                    "description": "My batch of recipes", 
+                    "status": "SUBMITTED", 
+                    "recipe_type": 6, 
+                    "event": 19, 
+                    "creator_job": 62, 
+                }, 
+                "is_superseded": true, 
+                "superseded": "1970-01-01T00:00:00Z"
+            }, 
+            ... 
```

Comment:
Let's make the example more exact, so do a count of 1 (at the top) and remove and ellipses and extra comma.